app/views/trainees/show.html.erb

```diff
@@ -1,59 +1,86 @@
-<%= form_with url: trn_submissions_path, method: :post, local: true do |f| %>
-  <%= hidden_field_tag :trainee_id, @trainee.id %>
-  <%= f.govuk_submit "Submit for TRN" %>
-<%- end -%>
 
-<h2 class="govuk-heading-l">Personal details</h2>
+<%= content_for(:breadcrumbs) do %>
+  <%= render_component GovukComponent::BackLink.new(
+    text: 'All records',
+    href: trainees_path
+  ) %>
+<% end %>
 
-<p class="govuk-body">
-  <%= govuk_link_to "Update", personal_details_trainee_path(@trainee) %>
-</p>
+<h1 class="govuk-heading-l">
+  <span class="govuk-caption-xl">Add a trainee</span>
```

Comment:
Should be `govuk-caption-l`

I'm not convinced this needs to be within the h1.

Reply:
Good catch but I think the caption needs to be inside the h1 to give screen reader users more context about what "Overview" is if they are using headings for in page navigation

Reply:
Ok, sounds good!